panels/dev-service/ha-panel-dev-service.html

```diff
@@ -49,25 +59,44 @@
         </app-toolbar>
       </app-header>
 
+      <app-localstorage-document
+        key='panel-dev-service-state-domain'
+        data='{{domain}}'>
+        <app-localstorage-document
+          key='panel-dev-service-state-service'
+          data='{{service}}'>
+          <app-localstorage-document
+            key='panel-dev-service-state-servicedata'
+            data='{{serviceData}}'>
+          </app-localstorage-document>
+        </app-localstorage-document>
+      </app-localstorage-document>
+
       <div class$='[[computeFormClasses(narrow)]]'>
         <div class='flex'>
           <p>
             Call a service from a component.
-          </p>
 
+          </p>
+          <div class='ha-dropdown'>
+            <paper-dropdown-menu class='dropdown-menu' label='Available Services' focused>
+              <paper-listbox class='dropdown-content'>
+                <template is='dom-repeat' items='[[computeDomains(serviceDomains)]]' as='domain'>
+                  <template is='dom-repeat' items='[[computeServices(serviceDomains, domain)]]' as='service'>
+                    <paper-item on-tap="serviceSelected">{{domain}}/{{service}}</paper-item>
+                  </template>
+                </template>
+              </paper-listbox>
+            </paper-dropdown-menu>
+          </div>
           <div class='ha-form'>
-            <paper-input label="Domain" autofocus value='{{domain}}'></paper-input>
-            <paper-input label="Service" value='{{service}}'></paper-input>
-            <paper-textarea label="Service Data (JSON, optional)" value='{{serviceData}}'></paper-textarea>
+            <paper-input label='Domain' value='{{domain}}'></paper-input>
+            <paper-input label='Service' value='{{service}}'></paper-input>
+            <paper-textarea label='Service Data (JSON, optional)' value='{{serviceData}}'></paper-textarea>
             <paper-button on-tap='callService' raised>Call Service</paper-button>
           </div>
           <div class='description'>[[description]]</div>
         </div>
-
-        <div>
-          <div class='header'>Available services</div>
-          <services-list on-service-selected='serviceSelected' hass='[[hass]]'></services-list>
```

Comment:
I think it's better to keep the services inside `services-list` component so it keeps this component manegable. There is no reason both should be merged.